### ISSUE TITLE
I Can not set minimum Month and Minimum year

### ISSUE THREAD

**Issue by islamarr** [lutvie72/RackMonthPicker#16]
How can I set the minimum Month and the Minimum year

**Comment by denihida1216** [comment]
on activity success tested call name activity.this, example:
rackMonthPicker = new RackMonthPicker(MainActivity.this);

**Comment by snowaye** [comment]
> How can I set the minimum Month and the Minimum year

Yes, I also want to set minimum and maximum month and year. Please support this.
